_layouts/default.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-us">
   <head>
     <meta charset="UTF-8">
-    <title>{{ page.title | default: site.title }}</title>
+    <title>{{ site.title | default: site.github.repository_name }} by {{ site.github.owner_name }}</title>
```

Comment:
Any updates on this one? This is the only theme I saw that has broken titles. The others do the same thing as above (or leave of the `by {{ site.github.owner_name }}` part.